static/js/starred_messages.js

```diff
@@ -32,6 +32,17 @@ exports.count = function () {
     return exports.ids.num_items();
 };
 
+exports.get_starred_msg_ids = function () {
+    return exports.ids.keys();
+};
+
+exports.remove_all_stars = function () {
+
+    exports.ids.clear();
+
+    exports.rerender_ui();
+};
```

Comment:
These extra blank lines are bad style.  Though see my notes elsewhere that we should probably just delete this function.